bindings/nodejs/src/lib.rs

```diff
@@ -36,7 +36,10 @@ pub struct Operator(opendal::Operator);
 #[napi]
 impl Operator {
     #[napi(constructor)]
-    /// @see For a detailed definition of scheme, see https://opendal.apache.org/docs/category/services
+    /// @see For the full list of scheme, see https://docs.rs/opendal/latest/opendal/services/index.html
```

Comment:
Please add comments before `#[napi(constructor)]`.

And I want to avoid redirecting nodejs users to `docs.rs`. Maybe we can give a simple list of possible services based on enabled services?

Reply:
> Maybe we can give a simple list of possible services based on enabled services?

 It's a more generic need. We can deal with it in a slightly more uniform way.

Reply:
We may create an issue to track this.
I won't fix this in this PR.